aqi_monitor/requirements.txt

```diff
@@ -1,3 +1,4 @@
 celery==4.4.6
 Django==3.0.7
-requests==2.24.0
\ No newline at end of file
+psycopg2-binary==2.8.5
```

Comment:
Why don't we use a standard psycopg2 package?  
https://pypi.org/project/psycopg2/